.circleci/config.yml

```
@@ -13,17 +13,9 @@ workflows:
           name: "Test with SimpleSAMLphp 2.4.0"
           simplesamlphp_version: "2.4.0"
       - test-phpunit:
-          name: "Test with PHP 7.4"
-          php_version: "7.4"
-      - test-phpunit:
-          name: "Test with PHP 8.0"
-          php_version: "8.0"
-      - test-phpunit:
-          name: "Test with PHP 8.1"
-          php_version: "8.1"
-      - test-phpunit:
-          name: "Test with PHP 8.2"
-          php_version: "8.2"
+          matrix:
+            parameters:
+              php_version: ["7.4", "8.0", "8.1", "8.2", "8.3", "8.4"]
```

```suggestion
              php_version: ["7.4", "8.1", "8.2", "8.3", "8.4"]
```
Lets drop 8.0 as it is EoL (7.4 is as well, I know, but we have not declared incompatibility yet). There's probably an argument to be made for dropping something in the middle, too, but I'm fine with this as is.